extension/goplugin/schemas_test.go

```diff
@@ -436,11 +436,11 @@ var _ = Describe("Schemas", func() {
 			Expect(context).To(HaveKey("subobject"))
 			Expect(context["subobject"]).To(BeNil())
 		})
-		It("should convert resource to context with null string", func() {
+		It("should convert resource to context with null string defined", func() {
 			resource := &test.Test{
 				ID:          "42",
 				Description: "test",
-				Name:        goext.MakeNullString("testName"),
+				Name:        goext.MakeMaybeString("testName"),
```

Comment:
MakeMaybeString and MakeNullString are inconcise for me. Once there is Maybe word and otherwise not.
I'd suggest naming:
MakeString / MakeNullString / MakeUndefinedString